scalatest/src/main/scala/org/scalatest/fixture/AsyncWordSpecLike.scala

```diff
@@ -50,7 +50,7 @@ StringVerbBlockRegistration, SubjectWithAfterWordRegistration}
  */
 //SCALATESTJS-ONLY @scala.scalajs.js.annotation.JSExportDescendentClasses(ignoreInvalidDescendants = true)
 @Finders(Array("org.scalatest.finders.WordSpecFinder"))
-trait AsyncWordSpecLike extends AsyncTestSuite with AsyncTestRegistration with ShouldVerb with MustVerb with CanVerb with Informing with Notifying with Alerting with Documenting { thisSuite =>
+trait AsyncWordSpecLike extends AsyncTestSuite with AsyncTestRegistration with ShouldVerb[Future[PendingStatement]] with MustVerb[Future[PendingStatement]] with CanVerb[Future[PendingStatement]] with Informing with Notifying with Alerting with Documenting { thisSuite =>
 
```

Comment:
In addition we could potentially move pending and pendingUntilFixed out of Assertions (via a deprecation cycle). Put pending into trait/object Pending and pendingUntilFixed into trait/object PendingUntilFixed, and then just add an AsyncPendingUntilFixed that looks for Future[PendingStatement]. That would simplify the signatures of the two pendingUntilFixed methods.